### PR TITLE
Scripts

### PR DESCRIPTION
**Pull request by AurumTheEnd**

### CODE REVIEW

File: start.bat [hunk 1]
```diff
@@ -111,3 +102,19 @@ set CLASSPATH=%TOOLS%;%RSYNTAXTEXTAREA%;%GSON%;%BYTEMAN%;%JRD%;%CPLC%
 if "%PORTABLE_JRD_HOME:~0,2%"=="//" (
   popd
 )
+
+rem Exit before function section
+exit /B 0
+
+rem Function for finding mvn repo files (2nd arg) of a certain group (1st arg) & returning it through the 3rd arg
+:findLib
```
Comment (judovana) [inline start.bat:112]: This works? You can really ahve function declaration after usage in bats?

File: .github/workflows/maven.yaml [hunk 2]
```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Build CPLC with Maven
         run: |
           cd cplc
-          mvn --batch-mode --update-snapshots clean install -DskipTests
+          mvn --batch-mode --update-snapshots clean install
```
Comment (judovana) [inline .github/workflows/maven.yaml:38]: I'm not sure with this. Released cplc should be used. Not random build

File: start.bat (hunk 1 above)
Comment (AurumTheEnd) [reply start.bat:112]: To answer your first question, yes it works. Both in development and in a portable image.

File: .github/workflows/maven.yaml (hunk 2 above)
Comment (judovana) [reply .github/workflows/maven.yaml:38]: pls remove the cplc build from cli. Otherwise this PR is ok to go

**Pull request merged**